Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -664,54 +669,70 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     // MARK: - Tunnel Stop
 
-    open override func stopTunnel(with reason: NEProviderStopReason, completionHandler: @escaping () -> Void) {
```

Comment:
Below this point the diff looks ugly so I'll explain the changes.

I'm changing our `stopTunnel` and `cancelTunnel` methods to be async, which allows us to track failures more easily for firing pixels.  If you see the full file at this point you'll see the new code, which IMO is much cleaner.